Zainab Saud/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
```

Comment:
liked you are using FontAwesome.
also read this article about the placement of `script` tag
https://www.digitalocean.com/community/tutorials/how-to-add-javascript-to-html